Pipfile

```diff
@@ -34,9 +34,10 @@ celery = {extras = ["redis"],version = "==5.0.4"}
 structlog = "==20.1.0"
 uritemplate = "==3.0.1"
 pyyaml = "==5.3.1"
+docker-compose = "*"
```

Comment:
@jycordero I'd rather not include docker-compose here, but I might be missing some more context on _why_.